packages/lib/server/queries/membership.ts

```diff
@@ -0,0 +1,109 @@
+import prisma from "@calcom/prisma";
+
+export async function isTeamAdmin(userId: number, teamId: number) {
+  const team = await prisma.membership.findFirst({
+    where: {
+      userId,
+      teamId,
+      accepted: true,
+      OR: [{ role: "ADMIN" }, { role: "OWNER" }],
+    },
+    include: {
+      team: {
+        select: {
+          metadata: true,
+          parentId: true,
+          isOrganization: true,
+        },
+      },
+    },
+  });
+  if (!team) return false;
+  return team;
+}
+
+export async function isTeamOwner(userId: number, teamId: number) {
+  return !!(await prisma.membership.findFirst({
+    where: {
+      userId,
+      teamId,
+      accepted: true,
+      role: "OWNER",
+    },
+  }));
+}
+
+export async function isTeamMember(userId: number, teamId: number) {
+  return !!(await prisma.membership.findFirst({
+    where: {
+      userId,
+      teamId,
+      accepted: true,
+    },
+  }));
+}
```

Comment:
@eunjae-lee would you be down to use the functions we already have for checkTeamAdmin etc - makes my life migrating to PBAC a bit easier